koala/ast/graph.py

```diff
@@ -376,11 +417,14 @@ def evaluate(self,cell,is_addr=True):
             if e.message.startswith("Problem evalling"):
                 raise e
             else:
-                # print "zzzzz", self.eval_ref("Cashflow!L136","Cashflow!DG136")
+                # print "PB 1", Range.apply_all('multiply',self.eval_ref("Calculations!P58:P68"),self.eval_ref("Calculations!P91:P101"),('124', 'P'))
                 raise Exception("Problem evalling: %s for %s, %s" % (e,cell.address(),cell.python_expression)) 
 
         try:
-            return cell.value
+            if type(cell.value) == Range:
+                return cell.value.values()
+            else:
```

Comment:
can we update value property to directly return values() if the cell contains a Range ?